extensions/wikia/ArticleExporter/ArticleExporter.setup.php

```diff
@@ -0,0 +1,14 @@
+<?php
+/**
+ * Article Exporter Extension
+ *
+ * @author Lore team
+ */
+
+$dir = __DIR__ . '/';
+
+// Autoload
+$wgAutoloadClasses['ArticleExporterHooks'] = dirname( __FILE__ ) . "/ArticleExporterHooks.class.php";
+
+// Hooks
+$wgHooks['ArticleSaveComplete'][] = 'ArticleExporterHooks::onArticleSaveComplete';
```

Comment:
I wonder if we should gate this hook behind a global so that we can turn it off without affecting the API. What do you think?